www-client/kristall/kristall-0.7.3~git.recipe

```diff
@@ -0,0 +1,68 @@
+SUMMARY="A cross-platform, multi-protocol visual browser for the small-internet"
+DESCRIPTION="Kristall is a browser without support for css/js/wasm or graphical \
+websites. It can display user-styled documents in several formats, \
+including Gemini, HTML, and Markdown provided by a server via Gemini, \
+Gopher, HTTP/HTTPS, and Finger protocols."
+HOMEPAGE="https://kristall.random-projects.net/"
+COPYRIGHT="2020 Felix Queißner"
+LICENSE="GNU GPL v3"
+REVISION="1"
+srcGitRev="b684f94f1af9a19c1a6fd70d72097a13b75e1ca6"
```

Comment:
Any specific reason to use srcGitRev here? Latest official release is from June this year: https://github.com/MasterQ32/kristall/releases/tag/V0.3

Reply:
Mostly to just have the package include the latest changes between the current release and the next one. I could change it to be based on the latest tagged release if that's more preferable.